resources/css/privacy.css

```
@@ -0,0 +1,102 @@
+	--privacy-reconsent-bg: #fef3c7;
+	--privacy-reconsent-text: #111827;
+
+	/* Policy page */
+	--privacy-toc-bg: #f8fafc;
+	--privacy-toc-text: #1f2937;
+	--privacy-body-text: #111827;
+	--privacy-link: #1d4ed8;
+}
+
+:root[data-theme="dark"],
+@media (prefers-color-scheme: dark) {
```

Stylelint (17.13.0)

[error] Unknown type selector "`@media`" (selector-type-no-unknown)

(selector-type-no-unknown)